src/main/java/com/wintertodt/scouter/ui/condensed/WintertodtScouterCondensedPluginPanel.java

```diff
@@ -276,14 +276,54 @@ private int getCompareValue(WintertodtScouterTableRow row1, WintertodtScouterTab
 	@Override
 	public void populate(List<WintertodtBossData> globalBossData)
 	{
-		rows.clear();
+		int currentWorld = plugin.getCurrentWorld();
```

Comment:
nit: let's move this right above the `for` loop declaration so it's closer to where it's being used.